src/components/CCard.vue

```diff
@@ -48,7 +48,9 @@
       ref="cardTextComponent"
       class="ma-0 px-0"
       :class="
-        scrollingContentVerticalPadding && scrollingEnabled ? 'py-4' : 'py-0'
+        scrollingContentVerticalPadding && scrollingEnabled
+          ? 'pt-4 pb-0'
+          : 'py-0'
```

Comment:
I don't quite understand, but isn't the bottom padding 0 in both cases? How does that fit the commit message
> fix: remove bottom padding of card content when scrollable

Reply:
no, before the change, the padding was `py-4`, if I understand you correct

Reply:
Yeah, now that you say it, it does actually make sense...